Базовая стилизация всех страниц

ParovatkinA opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/302695/tasks/6)

:boom: https://htmlacademy-htmlcss.github.io/302695-sedona-33/5/

ParovatkinA committed:
создает стилевой файл styles.css

ParovatkinA committed:
подключает нестандартные шрифты

из папки fonts со свойством font-display: swap

ParovatkinA committed:
подключает стилевой файл к главной странице

ParovatkinA committed:
Добавим фоновый цвет и цвет текста для шапки

ParovatkinA committed:
добавляет шрифт, цвет и цвет фона навигации в шапке

ParovatkinA committed:
добавляет остальные свойства в CSS по макету

ParovatkinA committed:
добавляет общую стилизацию кнопок

ParovatkinA committed:
создает блок с классом first-screen

для стилизации <h2> и <р> (СЕДОНА НЕБОЛЬШОЙ ГОРОДОК...)

ParovatkinA committed:
создает новый класс для преимуществ №1 и №5

ParovatkinA committed:
добавляет лого-картинку .htmlacademy

ParovatkinA committed:
добавляет базовую стилизацию страницы каталога

ParovatkinA committed:
добавляет стилизацию кнопки button для страницы каталога

ParovatkinA committed:
подключает стили к странице каталога

ParovatkinA committed:
добавляет стили для пагинации, страницы каталога и блока рассылки

ParovatkinA committed:
добавляем div для блока сортировки гостиниц

ParovatkinA committed:
добавляет новый класс для блока рассылки на странице catalog.html

ParovatkinA committed:
добавляет картинку logo htmlacademy

mpodaniev commented on styles/styles.css:
```diff
@@ -0,0 +1,305 @@
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+  }
+  
+  @font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 700;
+    src: url("../fonts/ptsans-700.woff2") format("woff2");
+    font-display: swap;
+  }
```
У первого селектора нет отступа, а у второго уже есть. Это нужно делать если есть вложенность, но здесь вложенности нет, поэтому отступа так же не должно быть.

mpodaniev replied:
```suggestion
@font-face {
    font-family: "PT Sans";
    font-style: normal;
    font-weight: 400;
    src: url("../fonts/ptsans-400.woff2") format("woff2");
    font-display: swap;
  }
  
@font-face {
    font-family: "PT Sans";
    font-style: normal;
    font-weight: 700;
    src: url("../fonts/ptsans-700.woff2") format("woff2");
    font-display: swap;
  }
```

mpodaniev commented on styles/styles.css:
```diff
@@ -0,0 +1,305 @@
+    src: url("../fonts/ptsans-700.woff2") format("woff2");
+    font-display: swap;
+  }
+  body {
+    font-family: "PT Sans", sans-serif;
+    font-size: 18px;
+    line-height: 21px;
+    color: #333333;
+    background-color: #ffffff;
+  }
+  .button {
+    font-family: inherit;
+    font-size: 20px;
+    line-height: 26px;
+    font-weight: 700;
+    text-align: center;
+    color: #ffffff;
+    background-color: #83B3D3;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
```
Между селекторами нужно добавлять отступ
```suggestion
  body {
    font-family: "PT Sans", sans-serif;
    font-size: 18px;
    line-height: 21px;
    color: #333333;
    background-color: #ffffff;
  }
  
  .button {
    font-family: inherit;
    font-size: 20px;
    line-height: 26px;
    font-weight: 700;
    text-align: center;
    color: #ffffff;
    background-color: #83B3D3;
    text-decoration: none;
    text-transform: uppercase;
  }
```

mpodaniev commented on styles/styles.css:
```diff
@@ -0,0 +1,305 @@
+    text-align: center;
+    background-color: #756257;
+  }
+  /* Pages */
+/* Шапка */
```
Почему-то комментарии на разном уровне

mpodaniev commented:
1. Зачем здесь такие стили? По маке ту же это не нужно: 
![image](https://user-images.githubusercontent.com/8547830/152690246-700e4eca-5786-4954-b3e8-8f1f64ad84ce.png)

2. Цвет и текст не по макету:
![image](https://user-images.githubusercontent.com/8547830/152690287-1f5dba7d-3fdf-4fa2-ad88-e122e575fb8d.png)

3. Не задан фоновый цвет: 
![image](https://user-images.githubusercontent.com/8547830/152690382-dec2f9c5-9afc-476a-9b8f-16fdbb267f72.png)

4. Не задан фоновый цвет:
![image](https://user-images.githubusercontent.com/8547830/152690465-73340277-1368-44c4-a9b3-db0c1276556f.png)

5. Не совпадает фоновый цвет:
![image](https://user-images.githubusercontent.com/8547830/152690490-2d6cf733-1daa-4efa-bc61-91970f04c79d.png)

ParovatkinA committed:
добавляет отступы между селекторами

ParovatkinA committed:
убирает ненужные стили

добавила подчеркивание кнопки "показать карточки", чтобы выделить выбранную сортировку

ParovatkinA committed:
добавляет новый класс для выбранной карточки

изменяет цвет на зеленый

ParovatkinA committed:
изменяет цвет фона в форме ввода e-mail

ParovatkinA committed:
изменяет расположение блока div в index.html и добавляет для него цвет фона в CSS

ParovatkinA committed:
изменяет цвет фона кнопки "поиск гостиницы в седоне"

mpodaniev commented:
Все хорошо, только здесь еще нужно текст поправить:
![image](https://user-images.githubusercontent.com/8547830/152690287-1f5dba7d-3fdf-4fa2-ad88-e122e575fb8d.png)
